src/test/ui/lint/lint-const-item-mutation.stderr

```diff
@@ -0,0 +1,89 @@
+warning: attempting to modify a `const` item
+  --> $DIR/lint-const-item-mutation.rs:15:5
+   |
+LL |     ARRAY[0] = 5;
+   |     ^^^^^^^^^^^^
+   |
+   = note: `#[warn(const_item_mutation)]` on by default
+   = note: each usage of a `const` item creates a new temporary - the original `const` item will not be modified
+note: `const` item defined here
+  --> $DIR/lint-const-item-mutation.rs:11:1
+   |
+LL | const ARRAY: [u8; 1] = [25];
+   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
I would like the "^" to be cover the name and type, like this:
 ```suggestion
LL | const ARRAY: [u8; 1] = [25];
   | ^^^^^^^^^^^^^^^^^^^^
```

The reason is the const expression in RHS could be a big block.

Reply:
If we decide to do this, I think it should be done consistently (e.g. as a part of https://github.com/rust-lang/rust/pull/75465), rather than special-casing this lint.